releasenotes/AzureDevOpsPipelineExtension.md

```diff
@@ -0,0 +1,15 @@
+## Azure App Configuration Extension release notes
+
+The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version is printed out to console in the step of running the Azure App Configuration extension: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
+
+### 1.6.3
+* Fix case insensitive for keys in the Azure App Configuration extension side.
```

Comment:
This is vague. What does it mean to 'fix' case insensitivity? What changed, what's the new behavior?

Reply:
I actually was debating with myself whether I should put this update. I think it might make customers confusion here.

Reply:
Its good to have it if there is a noticeable behavior change. If it is completely hidden then sure, no need to talk about it.

Reply:
Yes, will remove this part.